tests/dids/genesis.testnet.did.json

```diff
@@ -1,51 +1,36 @@
 {
   "record": {
-    "@context": "https://w3id.org/veres-one/v1",
-    "id": "did:v1:test:nym:2pfPix2tcwa7gNoMRxdcHbEyFGqaVBPNntCsDZexVeHX",
+    "@context": [
+        "https://w3id.org/did/v0.11",
+        "https://w3id.org/veres-one/v1"
```

Comment:
not to full on lint the json, but the indent here is off.

Reply:
fixed.